File: Android/Sample/app.detail/src/main/java/net/wequick/example/small/app/detail/SubActivity.java
package net.wequick.example.small.app.detail;

import android.app.Activity;
import android.databinding.DataBindingUtil;
import android.os.Bundle;
import android.view.LayoutInflater;

import net.wequick.example.small.app.detail.databinding.ActivitySubBinding;

/**
 * Created by galen on 16/4/11.
 */
public class SubActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        ActivitySubBinding binding = DataBindingUtil.inflate(LayoutInflater.from(this), R.layout.activity_sub, null,false);
        setContentView(binding.getRoot());
        binding.setHandlers(new MyHandler());
    }
}

File: Android/Sample/app.detail/src/main/java/net/wequick/example/small/app/detail/SubActivity.kt
package net.wequick.example.small.app.detail

import android.app.Activity
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View

/**
 * Created by galen on 16/4/11.
 */
class SubActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val v = LayoutInflater.from(this).inflate(R.layout.activity_sub, null)
        setContentView(v)
        v.setOnClickListener { finish() }
    }
}
